batch/provision.yml

```diff
@@ -5,14 +5,13 @@
   gather_facts: False
   vars:
     # The following vars should probably be overridden
-    - name: ETL
+    - name: Lagunita 4.7.2_6 EMR Cluster
```

Comment:
I'm not a huge fan of the `_6` bit..
That's just the 6th iteration of this?
It's a bit confusing, given that the other numbers refer to an actual version number, but the suffix is arbitrary.